GUI/Main.cs

```diff
@@ -366,6 +366,13 @@ protected override void OnLoad(EventArgs e)
 
         protected override void OnFormClosing(FormClosingEventArgs e)
         {
+            //Only close the window, when the user has access to the "Exit" of the menu
+            if (!menuStrip1.Enabled)
```

Comment:
Just checking the logic here: if the menu strip *isn't* enabled, then closing the window is treated as a cancel instead?